spring-data-neo4j-rx/src/main/java/org/neo4j/springframework/data/repository/support/SimpleNeo4jRepository.java

```diff
@@ -265,7 +265,7 @@ public boolean existsById(ID id) {
 	}
 
 	@Override
-	public Iterable<T> findAll() {
+	public List<T> findAll() {
```

Comment:
Shouldn't all `findAll...` methods return a `List`?

Reply:
Crap. That went into the PR by coincide.